fdbclient/NativeAPI.actor.cpp

```diff
@@ -3005,6 +3005,11 @@ void Transaction::setOption( FDBTransactionOptions::Option option, Optional<Stri
 			}
 			break;
 
+		case FDBTransactionOptions::SET_AS_DEBUG_TRANSACTION:
+			validateOptionValue(value, false);
```

Comment:
I wonder if it would be helpful to take an arbitrary string as an argument such that we could log the generated ID along with the passed in argument and later piece together who was the originator of a transaction sequence. Or I suppose would could instead use the value set in the `debug_transaction_identifier` to relate the generate UID with a custom string.

Reply:
Here's my thinking on the API: https://github.com/apple/foundationdb/issues/2644#issuecomment-592661212 It's probably worth keeping the discussion together there.